### ISSUE TITLE
NVDA switches to focus mode in some tables,

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5464]
**Reported by gregjozk_ on 2015-11-05 01:27**
Hi,

I can not find appropriate ticket, so I opened a new one.

In some last versions of Next brunch (probably, when working with Mozilla has been improved) NVDA switches to focus mode even I want to use a browse mode.

See:

www.cobiss.si (also in english) and one of slovenian webstores www.enaa.com

steps to reproduce a problem:
- Open Mozilla Firefox (in IE I did not found this problem).
- into first edit field type one string
- go to table with search results and try to navigate between them during "automatic focus mode for carret movement is on.

When this option is off, it works correctly, but it means, that focus would not start when user comes to edit field, combobox or another element.

Hope, that this problem can be solved.

**Comment by bhavyashah** [comment]
Using Firefox 61.0.2 and NVDA 2018.3 Beta 3, I was unable to replicate this issue when making a search query of “a” on http://www.cobiss.si/. I will state that I have experienced jumpiness by NVDA acutely on Firefox and less prominently on Chrome when browsing heavy, large and dynamic webpages, including the random change to focus mode from browse mode. Having said that, the scope of this ticket is limited only to tables on Firefox, which was not only not reproducible right now to me but also overly specific. Since I am positive that there are other open issues dealing with NVDA’s jitteriness on some webpages more broadly, I would say that those more reflectively encompass the extent of this behaviour. In addition, since one of the provided test cases failed to exhibit the described behaviour at my end, Firefox and NVDA have had a lot of updates since the November of 2015, I would suggest safely closing this ticket for now. If the original author of this ticket can be contacted (I think gregjozk_ is still around on GitHub) or if somebody else can attest to this particular issue, we can always reopen and revisit this.

**Comment by gregjozk** [comment]
agree. may be I would change a title and description of this issue a little bit cause, NVDA switches to FM also on Wikipedia's or other tables, where you can decide, how elemenst of the table should be placed (sorted acending/decednding) sorted to date/place or other criterion. but it is logical so I agree to close it.

**Comment by ehollig** [comment]
Closing as requested in https://github.com/nvaccess/nvda/issues/5464#issuecomment-414120033. If you have an example where something like this still applies, please open a new issue.
